include/sdf/system_util.hh

```diff
@@ -27,7 +27,7 @@
 /** \def SDFORMAT_HIDDEN
  * Use to represent "symbol hidden" if supported
  */
-#define SDFORMAT_HIDDEN IGNITION_SDFORMAT_HIDDEN
+#define SDFORMAT_HIDDEN GZ_SDFORMAT_HIDDEN
```

Comment:
Oops, I think we're not ready to change this.

Reply:
agreed

Reply:
Interesting that the script edited one but not the other :thinking: Maybe I manually discarded one of them and forgot to discard the other. Oops.

https://github.com/ignitionrobotics/sdformat/pull/1008/commits/b751698896d4c9d87e09d6cd09d8dcff5e54b5ad